src/main/java/com/marklogic/client/dataloader/impl/DefaultSchemasDataFinder.java

```diff
@@ -0,0 +1,29 @@
+package com.marklogic.client.dataloader.impl;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.marklogic.client.dataloader.SchemasDataFinder;
+
+/**
+ * Finds all files in a directory.
+ * TODO only select xsd, tde, and ruleset?
```

Comment:
One approach I've used elsewhere is to add a "private FileFilter schemasFileFilter" property on this class, and have it default to a sensible implementation. That gives an ml-gradle user enough flexibility to be able to swap out a different FileFilter if needed.